container/Dockerfile.centos

```diff
@@ -33,6 +33,7 @@ RUN mkdir -p $CONTRAIL && \
     echo export CONTRAIL=$CONTRAIL >> $HOME/.bashrc && \
     echo export LD_LIBRARY_PATH=$CONTRAIL/build/lib >> $HOME/.bashrc
 
-ADD Dockerfile entrypoint.sh $HOME/
+ADD entrypoint.sh $HOME/
+ADD Dockerfile.centos $HOME/Dockerfile
```

Comment:
should we copy Dockerfile at all now?
it was a hack - ADD command will fail if it has just one src and it's absent. but if it has multiple arguments and some is present - it will not fail.
I suppose that entrypoint.sh is always here - therefore we can remove copying of Dockerfile...

Reply:
ok if we don't need Dockerfile in there, I'll remove it. I figured it was added for a reason